apps/script-service/Services/LuaScriptService.cs

```diff
@@ -23,6 +23,37 @@ public LuaScriptService(ILogger<LuaScriptService> logger, IServiceDirectory dire
     _eventService = eventService;
   }
 
+  public IEnumerable<object> TestScript(
+    IDictionary<string, object?> inputs,
+    string script,
+    AdspId tenantId
+  )
+  {
+
+    _logger.LogDebug("Testing script for tenant {TenantId}...", tenantId);
+    try
+    {
+      using var lua = new Lua();
+
```

Comment:
Need this for the `adsp` global variable to be set.
```csharp
lua.RegisterFunctions(...)
```
Let's create a stub class with no functions to start to use instead of `ScriptFunctions`.

Reply:
Hi @tzuge, I added lua.RegisterFunctions(null) in the TestScript function.